fdbserver/include/fdbserver/KmsConnectorInterface.h

```diff
@@ -180,43 +174,20 @@ struct KmsConnLookupEKsByDomainIdsRep {
 	}
 };
 
-struct KmsConnLookupDomainIdsReqInfoRef {
-	constexpr static FileIdentifier file_identifier = 8980149;
-	EncryptCipherDomainId domainId;
-	EncryptCipherDomainNameRef domainName;
-
-	KmsConnLookupDomainIdsReqInfoRef() : domainId(INVALID_ENCRYPT_DOMAIN_ID) {}
-	explicit KmsConnLookupDomainIdsReqInfoRef(Arena& arena, const KmsConnLookupDomainIdsReqInfoRef& from)
-	  : domainId(from.domainId), domainName(StringRef(arena, from.domainName)) {}
-	explicit KmsConnLookupDomainIdsReqInfoRef(Arena& arena, const EncryptCipherDomainId dId, StringRef name)
-	  : domainId(dId), domainName(StringRef(arena, name)) {}
-	explicit KmsConnLookupDomainIdsReqInfoRef(const EncryptCipherDomainId dId, StringRef name)
-	  : domainId(dId), domainName(name) {}
-
-	bool operator==(const KmsConnLookupDomainIdsReqInfoRef& info) const {
-		return domainId == info.domainId && (domainName.compare(info.domainName) == 0);
-	}
-
-	template <class Ar>
-	void serialize(Ar& ar) {
-		serializer(ar, domainId, domainName);
-	}
-};
-
 struct KmsConnLookupEKsByDomainIdsReq {
 	constexpr static FileIdentifier file_identifier = 9918682;
 	Arena arena;
```

Comment:
Do we need this arena?

Reply:
Yeah, `RESTKmsConnector.actor.cpp::fetchEncryptionKeysByDomainIds` uses it to allocate JSON doc.